resources/run_adl.sh

```diff
@@ -0,0 +1,45 @@
+# 1. set git to use ssh instead of https
+git config --global url.git@github.com:.insteadOf https://github.com/
+
+# 2. clone the mongohouse repo
+# TODO - where to put it
+git clone git@github.com:10gen/mongohouse.git
+cd mongohouse
+
```

Comment:
Spawned a new host based on the evergreen run: ubuntu-1804.   The preinstalled go version was 1.7.  Will add suggestions to help install on that variant.
 ```suggestion

export GOVERSION=$(curl https://go.dev/VERSION?m=text)
wget "https://dl.google.com/go/${GOVERSION}.linux-amd64.tar.gz"
sudo rm -rf /opt/go && sudo tar -C /opt -xzf ${GOVERSION}.linux-amd64.tar.gz

```